cantabular/table.go

```diff
@@ -96,6 +129,60 @@ func GraphQLJSONToCSV(ctx context.Context, r io.Reader, w io.Writer) (int32, err
 	return rowCount, nil
 }
 
+// GraphQLJSONToJson converts a JSON response in r to a different schema on w, returning the response
+// if an error happens, the process is aborted and the error is returned.
+func GraphQLJSONToJson(ctx context.Context, r io.Reader, w io.Writer) (GetObservationsResponse, error) {
```

Comment:
Is there a unit test for this?

Reply:
This is fully tested as part of component tests in dp-population-types-api that consumes it.